src/devices/Pca95x4/samples/Pca95x4.Sample.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+using System.Device.I2c;
+using System.Device.I2c.Drivers;
+using System.Threading;
+
+namespace Iot.Device.Pca95x4.Samples
+{
+    class Program
+    {
+        private static readonly int s_deviceAddress = 0x38;
```

Comment:
For I2c devices, I think it could be useful if we start the pattern of having the default I2c address (and an alternate one in case the devices supports it) as static fields on the class, that way people will be able to easily tell which one to use without having to memorize it every time, what do you think?

I was thinking that if we did that, then people can consume this like:
```c#
var i2cDevice = new UnixI2cDevice(new I2cConnectionSettings(1, Pca95x4.DefaultI2cAddress));
var pca = new Pca95x4(i2cDevice);
```

Reply:
Yeah, this would be helpful in most cases and I'll add soon.  The user will need to be careful and check the datasheet (but most should be anyways 😄 ) as the default address could be different depending on the part.  For example, the PCA9554 base address is 0x20 and the PCA9554A base address is 0x38.

Reply:
I see, I just noticed that this is intended to support more than one device, in which case I agree that we should think a bit more about adding a default address. I still think though that it wouldn't hurt to provide one just for the default scenario to work when using the most common device, and then for others allow them to provide a different address. Anyways, this won't block this PR so I'll go ahead and merge this :)